ursula_cli/shell.py

```diff
@@ -25,6 +25,7 @@
 import subprocess
 from distutils.version import LooseVersion
 from ConfigParser import ConfigParser, NoOptionError, NoSectionError
+from heatclient.common import utils
```

Comment:
pep8 standards on imports are in order: standard lib, 3rd party modules, local modules/imports.  each group separated by a single line.  this import should go with 3rd party module imports by pep8.